### ISSUE TITLE
camunda-helm node option?

### ISSUE THREAD

**Issue by eins** [camunda-community-hub/camunda-helm#6]
has the camunda-helm a node option?
this means can I run camunda core and then in other pods camunda swagger, camunda rest.

**Comment by aabouzaid** [comment]
Hi @eins 
I believe that it is an application config, is it?

Could you please give more details about the exact option/config apart of the chart itself?

**Comment by eins** [comment]
I couldn't  find a way to pass the current camunda docker tag for run 7.15.0

```yaml
# image settings
image:
  repository: camunda/camunda-bpm-platform
  tag: run-7.15.0
metrics:
  enabled: false
```

It looks like camunda-helm only lunch a pod. I want to see if we can lunch three pods one for the camunda-core with the connection the the db and then two or three pods for webapps, rest and swaggerui

**Comment by aabouzaid** [comment]
@eins the new version of the chart v1.0.0 supports that via passing different args to the deployment.

You can see here the exact example where you can create rest only deployment:
https://github.com/camunda-community-hub/camunda-helm/tree/36a1ca6/charts/camunda-bpm-platform#example

**Comment by eins** [comment]
@aabouzaid thanks for the reply
Do you know if it is possible the change the image repository ?
camunda/camunda-bpm-platform

```
image:
  #  repository: camunda/camunda-bpm-platform
  repository: wwwwwww.dkr.ecr.eu-central-1.amazonaws.com/repo101:cee7.15.4
  tag: run
```

I got InvalidImageName and  ImagePullBackOff errors

**Comment by aabouzaid** [comment]
@eins you added the `tag` into `repository`
it should be like this, and you need to setup [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
```
image:
  repository: wwwwwww.dkr.ecr.eu-central-1.amazonaws.com/repo101
  tag: cee7.15.4
  pullSecrets:
    - DOCKER_CONFIG_JSON_SECRET
```

**Comment by aabouzaid** [comment]
Fixed by v1.0.0
